### PR TITLE
Correct a language error in reverse proxy doc

### PR DESCRIPTION
**Pull request by ZephOne**

### CODE REVIEW

File: docs/content/doc/usage/reverse-proxies.en-us.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ menu:
 ---
 
 ##  Using Nginx as a reverse proxy
-If you want Nginx to serve your Gitea instance, you can the following `server` section inside the `http` section of `nginx.conf`:
+If you want Nginx to serve your Gitea instance, you can follow the `server` section inside the `http` section of `nginx.conf`:
```
Comment (zeripath) [inline docs/content/doc/usage/reverse-proxies.en-us.md]:
```suggestion
If you want Nginx to serve your Gitea instance, add the following `server` section to the `http` section of `nginx.conf`:
```

**Review by 6543 — APPROVED** [review]
add your suggested change - but i dont block docs ;)

File: docs/content/doc/usage/reverse-proxies.en-us.md (hunk 1 above)
Comment (bagasme) [reply docs/content/doc/usage/reverse-proxies.en-us.md]:
```suggestion
If you want Nginx to serve your Gitea instance, add following `server` block inside the `http` section of `nginx.conf`. You can also add new configuration file (gitea.conf) and source it in `nginx.conf` by adding `include path/to/gitea.conf;`:
```

Comment (ZephOne) [reply docs/content/doc/usage/reverse-proxies.en-us.md]: So this is what was meant :smile:

**Review by zeripath — APPROVED** [review]

**Review by lafriks — APPROVED** [review]

**Pull request merged**